Description: Full hardening doesn't work withou CPPFLAGS being taken into
 account
Author: Paul Gevers <elbrus@debian.org>

Index: libcleri/Release/src/subdir.mk
===================================================================
--- libcleri.orig/Release/src/subdir.mk
+++ libcleri/Release/src/subdir.mk
@@ -79,7 +79,7 @@ C_DEPS += \
 src/%.o: ../src/%.c
 	@echo 'Building file: $<'
 	@echo 'Invoking: Cross GCC Compiler'
-	gcc -I../inc -O3 -Wall $(CFLAGS) -c -fmessage-length=0 -fPIC -MMD -MP -MF"$(@:%.o=%.d)" -MT"$(@)" -o "$@" "$<"
+	gcc -I../inc -O3 -Wall $(CPPFLAGS) $(CFLAGS) -c -fmessage-length=0 -fPIC -MMD -MP -MF"$(@:%.o=%.d)" -MT"$(@)" -o "$@" "$<"
 	@echo 'Finished building: $<'
 	@echo ' '
 
